### PR TITLE
UI Component is now listed as a behaviour

### PR DESCRIPTION
**Pull request by jrbartola**

### CODE REVIEW

File: app/default-config.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "market_pairs": [],
     "loglevel": "",
     "update_interval": 300,
-    "selected_task": "default"
+    "selected_task": "server"
```
Comment (shads2) [inline app/default-config.json]: Do we want this to be the default behaviour? Does it sufficiently supersede the current default behaviour?

File: app/default-config.json [hunk 2]
```diff
@@ -66,6 +66,12 @@
         "hot": 0.8,
         "cold": 0.4
       }
+    },
+    "server": {
+
+    },
+    "server": {
```
Comment (shads2) [inline app/default-config.json]: The server key appears to be duplicated. I am uncertain why this doesn't break the JSON import as JSON implementations normally have big issues with duplicate keys.

Comment (jrbartola) [reply app/default-config.json]: Whoops, I'll remove that.

File: app/default-config.json (hunk 1 above)
Comment (jrbartola) [reply app/default-config.json]: Forgot to change that back after testing. Will update

**Pull request merged**